cluster-scope/overlays/prod/moc/smaug/kustomization.yaml

```diff
@@ -1,154 +1,150 @@
----
 apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
@GaryPWhite I think somewhere this file got reformatted again, can you fetch this from upstream and manually add the line: 

```
- ../../../../base/core/namespaces/tremor-demo
- ../../../../base/core/namespaces/ushift-dev
```

Alphabetically? 

It also seems like your rebase did not incorporate https://github.com/operate-first/apps/commit/422a3e8ab2951e9340d1052506d7562e7a2fca72

As it's attempting to remove:
```
- ../../../../base/rbac.authorization.k8s.io/clusterrolebindings/oauth-token-access-review-opf-observatorium
...
- ../../../../base/rbac.authorization.k8s.io/clusterroles/oauth-token-access-review

```